spec/02-integration/09-hybrid_mode/01-sync_spec.lua

```diff
@@ -353,272 +353,288 @@ for _, strategy in helpers.each_strategy() do
     end)
   end
 
-  describe("CP/DP version check works with #" .. strategy, function()
-    -- for these tests, we do not need a real DP, but rather use the fake DP
-    -- client so we can mock various values (e.g. node_version)
-    describe("relaxed compatibility check:", function()
-      local bp = helpers.get_db_utils(strategy, {
-        "routes",
-        "services",
-        "plugins",
-        "upstreams",
-        "targets",
-        "certificates",
-        "clustering_data_planes",
-      }) -- runs migrations
-
-      bp.plugins:insert {
-        name = "key-auth",
-      }
-      lazy_setup(function()
+  for _, cluster_protocol in ipairs{"wrpc", "json"} do
```

Comment:
let's not insert identions below so that we won't create lots of diffs and it will cause trouble when merging to ee